src/code/z_play.c

```diff
@@ -2311,18 +2312,18 @@ void Play_Init(GameState* thisx) {
     VisMono_Init(&sPlayVisMono);
 
     gVisMonoColor.a = 0;
-    D_801F6D4C = &D_801F6D38;
-    func_80140E80(D_801F6D4C);
-    D_801F6D4C->lodProportion = 0.0f;
-    D_801F6D4C->mode = 1;
-    D_801F6D4C->primColor.r = 0;
-    D_801F6D4C->primColor.g = 0;
-    D_801F6D4C->primColor.b = 0;
-    D_801F6D4C->primColor.a = 0;
-    D_801F6D4C->envColor.r = 0;
-    D_801F6D4C->envColor.g = 0;
-    D_801F6D4C->envColor.b = 0;
-    D_801F6D4C->envColor.a = 0;
+    sPlayVisFBufInstance = &sPlayVisFBuf;
+    VisFbuf_Init(sPlayVisFBufInstance);
+    sPlayVisFBufInstance->lodProportion = 0.0f;
+    sPlayVisFBufInstance->mode = 1;
```

Comment:
This could use one the mode enums? I think the general one.

Reply:
You mean the `TransitionMode` enum?

Reply:
I was thinking the `VisFBufMode` enum